### DISCUSSION TITLE
제페토 스튜디오 심사 제출 불가

### DISCUSSION THREAD

**Discussion by natural-jogi** [naverz/zepeto-studio-kor#42] [Issue Report (Legacy)]
현재 서버 문제가 있는건지 자체적으로 오류가 있는건지는 모르겠지만 심사 제출하기 버튼을 누르면 로딩 애니메이션만 잠깐 나오다가 아무런 메시지도 없이 심사 제출이 되지 않는 것 같습니다. 확인 부탁드립니다.

**Comment by cheolhui-lee** [comment]
안녕하세요.
혹시 패키지 파일의 업로드 단계를 노출하는 부분도 아무 변화가 없으실까요?
현재 상황의 스크린샷(이미지, 영상) 공유가 가능하시면 확인하는데 도움이 될 것 같습니다.
고맙습니다.
